app.js

```diff
@@ -190,6 +190,27 @@
       return undefined;
     }
   }
+  
+  function getAQIEmoji(aqi) {
+    if (aqi >= 401) {
+      return '&#x2620;'; // ☠
+    } else if (aqi >= 301) {
+      return '&#x1F635;'; // 😵
+    } else if (aqi >= 201) {
+      return '&#x1F922;'; // 🤢
+    } else if (aqi >= 151) {
+      return '&#x1F637;'; // 😷
+    } else if (aqi >= 101) {
+      return '&#x1F641;'; // ☹️
+    } else if (aqi >= 51) {
+      return '&#x1F610;'; // 😐
+    } else if (aqi >= 0) {
+      return '&#x1F600';  // 😀
+    } else {
+      return undefined;
```

Comment:
```suggestion
      return "";
```

I think we should just return empty string in the worst case. It's unlikely to happen, but there's no upside to putting an `undefined` string in the text we present.